terraform-setup/analytics/rds/main.tf

```diff
@@ -38,6 +38,12 @@ resource "aws_ssm_parameter" "rds_admin_password"{
   type = "String"
   value = random_password.rds_password.result
 }
+
+#Retrieve existing admin username created and saved in AWS SSM parameter store
+data "aws_ssm_parameter" "rds_admin_username"{
```

Comment:
@King-Juliet who is meant to use this RDS instance?
How would you name your ssm username if you have to create another RDS instance for another use case for example?

Reply:
The RDS instance is the company-wide source data. So everyone who is permitted to input data into the database, like, inventory personnel to update inventory table, HR personnel to update HR table, etc. The admin username and password is for the data engineer to make changes to the database structure of the database if the need arises.